src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DriverActionTranslator.java

```diff
@@ -8,11 +8,13 @@
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DriverActionTranslator extends DrawPanelController implements Job2dDriver {
```

Comment:
Interesting name, but I don't understand Translation part. The name does not indicate the purpose of the adapter or what was actually adapted.
Inheritance is not used (no super drawLine calls). Should be removed!